python/GafferSceneUI/SceneInspector.py

```diff
@@ -963,6 +969,15 @@ def __menuDefinition( self, widget = None ) :
 					}
 				)
 
+			if self.__inspector.supportsShaderParameters( target ) :
```

Comment:
Not something to worry about now, but I just want to mention it while we're here. At some point we'll need to abstract out the basics of the Inspector/Diff/DiffRow from the SceneInspector so they can form the basis of the ImageInspector too. At that point this `supportsShaderParameters()` concept isn't going to fit (and neither is `supportsInheritance()`). I think it's OK to keep hardcoding like this for now, but at some point I'm thinking this menu will need to be constructed more dynamically, using callbacks like we do for plug popup menus and so on.